app/scripts/controllers/transactions/index.test.js

```diff
@@ -2186,10 +2186,10 @@ describe('Transaction Controller', function () {
       assert.equal(result.userFeeLevel, 'high');
     });
 
-    it('throws error if status is not unapproved', function () {
+    it('should not update and should throw error if status is not unapproved', function () {
```

Comment:
again just a tiny nit for clarity (might just be for me haha)
```suggestion
    it('should not update and should throw error if status is not type "unapproved"', function () {
```

Reply:
Works for me! Thanks and updated